File: Ent_UsAdmin/CierraEmpresa.vb
Public Class CierraEmpresa
    Private Sub CierraEmpresa_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        System.Threading.Thread.Sleep(1000) ' 1 segundo
        SeCierra()

    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        frm_adm.cerrarEmpresa = True
        frm_adm.cerrarSesion = True
        frm_adm.Close()

    End Sub
    Private Sub SeCierra()
        frm_adm.cerrarEmpresa = True
        frm_adm.cerrarSesion = True
        frm_adm.Close()
    End Sub
End Class

File: Ent_UsAdmin/Acceso/CierraEmpresa.cs
using System;

namespace ConfiaAdmin
{
    public partial class CierraEmpresa
    {
        public CierraEmpresa()
        {
            InitializeComponent();
        }
        private void CierraEmpresa_Load(object sender, EventArgs e)
        {
            System.Threading.Thread.Sleep(1000); // 1 segundo
            SeCierra();

        }

        private void Button1_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.frm_adm.cerrarEmpresa = true;
            My.MyProject.Forms.frm_adm.cerrarSesion = true;
            My.MyProject.Forms.frm_adm.Close();

        }
        private void SeCierra()
        {
            My.MyProject.Forms.frm_adm.cerrarEmpresa = true;
            My.MyProject.Forms.frm_adm.cerrarSesion = true;
            My.MyProject.Forms.frm_adm.Close();
        }
    }
}